cmd.c

```diff
@@ -743,43 +743,52 @@ static int cmd_loadApp(phfs_conf_t *phfs, const char *name, const char *imap, co
 
 void cmd_app(char *s)
 {
-	u16 cmdArgsc = 0;
-	int i = 0, argID = 0;
+	u16 i, cmdArgsc = 0, argID = 0;
 	char cmdArgs[MAX_CMD_ARGS_NB][LINESZ + 1];
 
 	char *name;
-	unsigned int pos = 0;
+	unsigned int pos = 0, flags = 0;
 	char cmap[8], dmap[8];
 	char appData[3][LINESZ + 1];
 
-	phfs_conf_t phfs;
-
-	phfs.datasz = 0;
-	phfs.dataOffs = 0;
+	phfs_conf_t phfs = { 0 };
```

Comment:
For `TARGET=armv7a9-zynq7000` is an error
```
arm-phoenix-ld: /tmp/_build/armv7a9-zynq7000/plo/cmd.o: in function `cmd_app':
/tmp//plo/cmd.c:754: undefined reference to `memset'
```
With the optimization flag `-Os` ( this is used with `armv7a` ) the compiler uses the `memset()` function for zeroing.
You should go back to previous version.

Reply:
Thats right, `-Os` is present on zynq. Reverted to previous version, also because (structure `phfs_conf_t`) `dn` and `handle` are set later, there is no need to zero them, so the previous version (with assignment) is the quickest.